### PR TITLE
Implemented header section

### PR DESCRIPTION
**Pull request by Albinka-811**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                    <a class="header-menu-list-item__link">GIFTSET</a>
+                </li>
+                <li class="header-menu-list-item">
+                    <a class="header-menu-list-item__link">LIÊN HỆ</a>
+                </li>
+            </ul>
+        </div>
+    </header>
+</div>
+<script src="script.js"></script>
```
Comment (Anatolych97) [inline index.html]: You don't have script file at PR

Comment (Albinka-811) [reply index.html]: Fixed

File: src/styles/components/header.scss [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      border-radius: 50%;
+      position: absolute;
+      top: 50%;
+      left: 55%;
+      transform: translate(-50%, -50%);
+
+      &__icon {}
```
Comment (Anatolych97) [inline src/styles/components/header.scss]: Empty selector

File: src/styles/components/header.scss [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    }
+  }
+
+  &-menu {
+    background-color: $primary ;
+    &-cart {
```
Comment (Anatolych97) [inline src/styles/components/header.scss]: Add empty line before selectors

File: src/styles/variales.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+//colors
+$primary: #415167;
+$accent: #C7A17A;
+$darkGray: #151D28;
+$lightGray: #EDF0F5;
+$romance: #F9F5E8;
+$grey: #232C38;
+$white: #FFFFFF;
+
+//fond-size
```
Comment (Anatolych97) [inline src/styles/variales.scss]: Font-size

File: src/styles/main.scss [hunk 5]
```diff
@@ -0,0 +1,28 @@
+@import "variales.scss";
+@import "./components/header.scss";
+@import "./components/features.scss";
+@import "./components/slider.scss";
+@import "./components/gift-set.scss";
+@import "./components/combo-set.scss";
+@import "./components/footer.scss";
+@import "./components/tab.scss";
```
Comment (Anatolych97) [inline src/styles/main.scss]: Imports of non-existence files

File: src/styles/components/header.scss [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+      &__color {
+        color: $white;
+        background-color: $accent;
+      }
+    }
+  }
+
+  &-slide {
+    //position: relative;
```
Comment (Anatolych97) [inline src/styles/components/header.scss]: Comment

File: src/styles/components/header.scss [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+      top: 50%;
+      left: 55%;
+      transform: translate(-50%, -50%);
+
+      &__icon {}
+    }
+  }
+
+  &-menu {
+    background-color: $primary ;
```
Comment (Anatolych97) [inline src/styles/components/header.scss]: Redundant space before semicolon

File: src/styles/main.scss [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  background-color: #F9FBFF;
```
Comment (Anatolych97) [inline src/styles/main.scss]: Raw color value. Use variable

File: index.html [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="header-block">
+            <img class="header-block-logo" src="./src/assets/images/logo.png" alt="Логотип">
+            <h1 class="header-block-title">
+                YOUR<br>
+                <span class="header-block-title__color"> PERSONALIZED </span>
+                <br>COFFEE
+            </h1>
+        </div>
+        <div class="header-slide">
+            <img class="header-slide-photo" src="src/assets/images/image-caffe.png" alt="">
```
Comment (Anatolych97) [inline index.html:22]: Empty alt attr

File: src/styles/components/header.scss (hunk 2 above)
Comment (Albinka-811) [reply src/styles/components/header.scss]: fixed

File: src/styles/components/header.scss (hunk 3 above)
Comment (Albinka-811) [reply src/styles/components/header.scss]: fixed

File: src/styles/variales.scss (hunk 4 above)
Comment (Albinka-811) [reply src/styles/variales.scss]: remove comments

File: src/styles/main.scss (hunk 5 above)
Comment (Albinka-811) [reply src/styles/main.scss]: remove import files

File: src/styles/components/header.scss (hunk 6 above)
Comment (Albinka-811) [reply src/styles/components/header.scss]: remove comment

File: index.html (hunk 9 above)
Comment (Albinka-811) [reply index.html:22]: It is a decoration image

**Pull request merged**